gui-js/apps/minsky-electron/src/app/managers/ContextMenuManager.ts

```diff
@@ -1252,10 +1187,10 @@ export class ContextMenuManager {
     const r=await godley.popup.rowYZoomed(y);
     const c=await godley.popup.colXZoomed(x);
     const clickType = await godley.popup.clickTypeZoomed(x, y);
-    this.initContextMenuForGodley(godley, r, c, clickType, () => {});
+    this.initContextMenuForGodley(godley, r, c, clickType, () => CommandsManager.requestRedraw());
```

Comment:
Again - an arrow function is not necessary when you already have a named function object

Reply:
via a quirk of javascripts', the consequence of leaving out the () => ...() stuff is that the "this" of the class we're borrowing the method from remains bound to that method, that's why I usually prefer to write this out. I don't think it has any consequences here though, so your comment is fair.
Alternatively one can write CommandsManager.requestRedraw.bind(this) btw...